odd-platform-ui/src/components/Management/DataSourcesList/DataSourceForm/DataSourceForm.tsx

```diff
@@ -8,19 +8,20 @@ import {
   intervalToDuration,
 } from 'date-fns/esm';
 import { Controller, useForm } from 'react-hook-form';
+import { DataSource, DataSourceFormData } from 'generated-sources';
 import {
-  DataSource,
-  DataSourceApiRegisterDataSourceRequest,
-  DataSourceApiUpdateDataSourceRequest,
-  DataSourceFormData,
-} from 'generated-sources';
+  registerDataSource,
+  updateDataSource,
+} from 'redux/thunks/datasources.thunks';
```

Comment:
Please, check imports. Somewhere you import functions from `redux/thunks` or `redux/selectors`, but somewhere import path looks weird